telescope/selector.py

```diff
@@ -186,64 +185,35 @@ def parse_ip_translation(self, ip_translation_dict):
       raise ValueError('Missing expected field in ip_translation dict: %s' % e.args[0])
 
   def validate_selector_input(self, selector_dict):
-    if not selector_dict.has_key('file_format_version') or \
-            selector_dict['file_format_version'] < self.supported_file_format_versions['minimum'] or \
-            selector_dict['file_format_version'] > self.supported_file_format_versions['maximum']:
```

Comment:
I think we should keep this (but make supported version= `1.1, 1.1`).

The effect I see is that if we rev the version, and an old version of Telescope sees selectors for a newer version, it'll try to parse them anyway. In some cases that's fine like if we add a bufferbloat field in 1.2 and a selector specifies "bufferbloat=includeall", then 1.1 will just ignore it and get correct results. The pitfall is if it's "bufferbloat=do_special_thing_1.1_doesn't_know_about" then 1.1 will think it parsed correctly and get wrong results.